apps/studio/components/interfaces/Organization/TeamSettings/MembersView.tsx

```diff
@@ -410,3 +234,206 @@ const MembersView = ({ searchString }: MembersViewProps) => {
 }
 
 export default MembersView
+
+interface MemberRowProps {
+  member: OrganizationMember
+  roles: Role[]
+  isLoadingRoles?: boolean
+  showMfaEnabledColumn?: boolean
+  setUserRoleChangeModalVisible: Dispatch<SetStateAction<boolean>>
+  setSelectedMember: Dispatch<SetStateAction<SelectedMember | undefined>>
+}
+
+const MemberRow = ({
```

Comment:
we okay to just shift this into another file in the same folder? MemberRow.tsx
just to keep one file for one thing